src/components/data-table.jsx

```diff
@@ -42,18 +33,22 @@ const DataTable = (props) => {
         <tbody class={classNames(styles["table-body"], props.table_body_class)}>
           <For each={props.data}>
             {(cell_value, index) => (
-              <tr
-                draggable="true"
-                onTouchStart={() => setActiveIndex(cell_value.tick)}
-                ref={(el) => detectTouch(el)}
-              >
+              <tr>
                 <td>
                   <div
-                    class={classNames(styles["data-layout"], {
-                      [styles["slider--active"]]:
-                        swipe_direction() === "LEFT" &&
-                        cell_value.tick === active_index(),
-                    })}
+                    class={classNames(
+                      styles["data-layout"],
+                      {
+                        [styles["data-layout--fav"]]: isFavourites(
+                          props.config.ref
+                        ),
+                      },
+                      {
+                        [styles["data-layout--market"]]: !isFavourites(
+                          props.config.ref
+                        ),
+                      }
+                    )}
```

Comment:
cant we add both the style conditions in a single object?

Reply:
I have made them into a single object